tests/rados/test_osd_replicated_inconsistency_scenario.py

```diff
@@ -0,0 +1,219 @@
+"""
+This file contains the  methods to verify the  inconsistent object functionality during scrub/deep-scrub
+in replicated pool.
+AS part of verification the script  perform the following tasks-
+   1. Creating omaps
+   2. Convert the object in to inconsistent object
+   3. Verify the functionality during scrub/deep-scrub
```

Comment:
We are mostly verifying the behaviour of "osd_scrub_auto_repair_num_errors" & "osd_scrub_auto_repair" params. Please add details on how these params are verified in the workflow.

Reply:
Added the Doc text.